.eslintrc.js

```diff
@@ -16,8 +16,10 @@ module.exports = {
 		"no-undef": "error",
 		"no-extra-semi": "error",
 		"semi": "error",
+		"quotes": ["error", "double", { "avoidEscape": true }],
 		"no-template-curly-in-string": "error",
 		"no-caller": "error",
+		"dot-notation": "error",
```

Comment:
Actually it does not improve readability:

```js
// easy to read
// visual consistency
headers["Context-Type"] = contentType;
headers["Location"] = url;

headers["Context-Type"] = contentType;
headers.Location = url;
//     ^
```

Reply:
In those scenario you can disable the rule via comment or if there are global used variables we can use `allowPattern` rule option.

At the moment the brackets syntax is used only in `test/Parser.unittest.js` so I added disable comment.

If you don't like this option I will remove the rule.